docs/changelog/changelog_en.md

```diff
@@ -1,5 +1,78 @@
 # Changelog plentyshopPWA
 
+## v1.4.0 (2024-04-15) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0" target="_blank" rel="noopener">
+
+### Migration guide
+
+- The upload action was changed [.github/workflows/upload.yml](https://github.com/plentymarkets/plentyshop-pwa/compare/v1.3.0...v1.4.0#diff-8cf0b5fae548aab9bd49f5020da9c0e35d281984b82b8a5358ffb1c1ae5bec13L5) update the file to make use of the [config feature](https://pwa-docs.plentymarkets.com/guide/setup/deployment#config)
+- We now require an access token to make requests to the plentysystems API. [guide](todo)
```

Comment:
Would be good if we could add the correct name here. I don't think it is called access token right? Also the Guide inside the documentation needs to be linked here!